UnicornEngine/sources/video/Renderer.cpp

```diff
@@ -396,7 +334,7 @@ void Renderer::onWindowResized(GLFWwindow* window, int width, int height)
 
 bool Renderer::RecreateSwapChain()
```

Comment:
From what I can see this method and other CreateX methods always return true and do not need to be bool.